app/Filament/Customer/Resources/CustomerCreditNotes/Tables/CustomerCreditNotesTable.php

```
@@ -0,0 +1,53 @@
+use Filament\Actions\ViewAction;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+
+class CustomerCreditNotesTable
+{
+    public static function configure(Table $table): Table
+    {
+        return $table
+            ->defaultSort('created_at', 'desc')
+            ->query(
+                CustomerCreditNote::where('client_id', auth()->user()->contact->third_party_id)
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Commerce\CustomerCreditNote::where().


(staticMethod.notFound)